libs/cms/src/lib/loaders/organizationLogoByKey.loader.ts

```diff
@@ -0,0 +1,36 @@
+import DataLoader from 'dataloader'
+import { Injectable } from '@nestjs/common'
+
+import { NestDataLoader } from '@island.is/nest/dataloader'
+import { CmsContentfulService } from '../cms.contentful.service'
+
+type LogoUrl = string | null
+
+export type OrganizationLogoByKeyDataLoader = DataLoader<
```

Comment:
I'm wondering if it makes more sense to call this "ByReferenceId" or "ByReferenceIdentifier" since key is ambiguous

Reply:
yeah makes sense, I changed it

Reply:
Alright, perhaps renaming the file as well makes sense?